spec/middleware-1.0.0-draft.1.md

```diff
@@ -0,0 +1,309 @@
+# OWIN Middlewares
+
+**Version**
+1.0.0-draft.1
```

Comment:
Is this `1.0.0`, `1.0.1`, or `1.1`? Middleware is currently under the `1.1` version. I can't recall exactly which versioning strategy we are following.